cardano-node/ChangeLog.md

```diff
@@ -12,10 +12,14 @@ None
 
 ### network changes
 
-- New P2P topology file format. See [issue #4563][#4563] or the [config
-  files][understanding-config-files] documentation. The old P2P topology
+- Added 'DemoteLocalAsynchronous' warning trace.  It indicates that a remote
```

Comment:
Should this be `RemoteLocalAsynchronous`? Also why are you deleting "New P2P topology file format..."?

Reply:
`Demote` is intentional here and the P2P part got pushed down (for some reason the differ got confused here).